packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.tsx

```diff
@@ -17,11 +16,10 @@ export class PrimaryButton extends BaseComponent<IButtonProps, {}> {
     const { theme, styles } = this.props;
 
     return (
-      <BaseButton
+      <DefaultButton
         { ...this.props }
-        variantClassName='ms-Button--primary'
```

Comment:
from a design standpoint, standard and primary button are 2 variants of the same button. The goal of this PR is to get the code in a similar state.

As you can see, all buttons have standard and primary style variation. 

![image](https://user-images.githubusercontent.com/1434956/29329525-fee709aa-81aa-11e7-81ef-4ca51d774f4c.png)